test/e2e/bindata/assets/04_descheduler-cluster-role-binding.yaml

```diff
@@ -1,12 +1,12 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: openshift-descheduler-operator
+  name: openshift-descheduler
```

Comment:
Is this meant to be for the operand or the operator?

Reply:
That's why this repository is confusing :sweat_smile: Things sometimes are called as simply "descheduler" and sometimes as "descheduler-operator" and it is not clear to which things are actually referring.